samples/bluetooth/peripheral_mds/sample.yaml

```diff
@@ -7,6 +7,8 @@ tests:
     extra_configs:
       - CONFIG_MEMFAULT_NCS_PROJECT_KEY="dummy-key"
       - CONFIG_MEMFAULT_NCS_DEVICE_ID="dummy-device-id"
+      # Needed to avoid warning from Memfault SDK
+      - CONFIG_LOG_PRINTK=n
```

Comment:
Is it ok to disable this config only in this place?

Reply:
We just shipped a fix in v1.3.3 of the Memfault SDK (see Changlog [here](https://github.com/memfault/memfault-firmware-sdk/blob/master/CHANGES.md#chart_with_upwards_trend-improvements)) addressing the warning, so this extra config can be removed as long as the sdk version is bumped to 1.3.3.

Reply:
Thanks, I have updated the version to 1.3.3 and removed the workaround